client/components/forms/docs/example.jsx

```diff
@@ -77,10 +82,14 @@ var FormFields = React.createClass( {
 
 					<FormFieldset>
 						<FormLegend>Form Checkbox</FormLegend>
-						<FormLabel>
-							<FormCheckbox id="comment_like_notification" name="comment_like_notification" />
-							<span>Email me when someone Likes one of my comments.</span>
-						</FormLabel>
+
+						<Checkbox
+							id="comment_like_notification"
```

Comment:
>I see your point, but I tried to be explicit.

Yep, no need to change anything here. Thinking out loud a bit about when or if it's ever appropriate to use the props spread :smile: 

Thanks for the information about the need for `name`.